wandb/sdk/data_types/table.py

```diff
@@ -480,6 +480,11 @@ def _to_table_json(self, max_rows=None, warn=True):
             max_rows = Table.MAX_ROWS
         n_rows = len(self.data)
         if n_rows > max_rows and warn:
+            # NOTE: Never raises for reinit="create_new" runs.
```

Comment:
so tables won't work correctly in the world where we have create_new right? since it will send way above the max allowed number of rows, i'm not sure what happens in the backend in that case

Reply:
There will still be a limit, but it will always result in a warning and never a ValueError.